repos/spack_repo/builtin/packages/xyce/package.py

```diff
@@ -103,6 +104,14 @@ class Xyce(CMakePackage):
         description="Require static blas build for PyMi",
     )
 
+    variant("fftw", default=True, description="Depend on FFTW")
+    depends_on("fftw~mpi", type=("build", "run"), when="+fftw~mpi")
+    depends_on("fftw+mpi", type=("build", "run"), when="+fftw+mpi")
+
+    # https://github.com/Xyce/Xyce/commit/ddec31a9c42c683831937be17fd6ffc3180e77a1
+    # requirement because of use of std::filesystem
+    conflicts("@7.10:", when="%gcc@:8")
```

Comment:
Did you mean for this to be `@7.9:` (i.e., the version you're adding)?

Reply:
Hi @tldahlgren, thanks for taking a detailed look. Yes, I did mean for it to be `@7.10:` rather than `@7.9:`. 7.9 was released in early February of 2025 and the commit in question (that the conflict is added in relation to) was added in March. That commit will be part of 7.10 which is currently in pre-release and I expect will be released soon. It seemed better to anticipate 7.10 with this conflict that also will work for git branches rather than making one for `@master` and then changing it to `@7.10:` in a few weeks.